code/datums/gamemode/factions/bloodcult/bloodcult_items.dm

```diff
@@ -144,10 +144,12 @@
 	autoignition_temperature = AUTOIGNITION_PAPER
 	fire_fuel = 1
 
+/*
```

Comment:
why comment this out? Seems like a fun jokey way of being turned mid-round cult, like the stone mask, syndicate uplink, and changeling vial.

'Hey, want to join our book club? Just look at this icon and all will become clear'

Reply:
That was just a debug item to test the cult spells. I'm planning to re-use the sprite as a base for new talismans.

I'm not excluding (in fact I've already considered) items that can turn you mid-round into a cultist but I'm planning something more elaborate and interesting than just "touch this paper, you're a cultist now!"

Reply:
okay